corehq/apps/app_manager/static/app_manager/js/forms/case_config_ui.js

```diff
@@ -317,14 +317,33 @@ hqDefine('app_manager/js/forms/case_config_ui', function () {
                 self.case_name = null;
             }
 
+            self.sortProperties = function (p1, p2) {
+                var validPaths = [];
```

Comment:
These declarations and the `_.each` could be a one-liner: `var validPaths = _.pluck(caseConfig.questions(), 'value')`

Reply:
Ooo handy, changed!

Reply:
I'm jealous. I've always wanted to use `pluck`, but have never thought of it if I ever had the chance.